extension/src/pipeline/quickPick.ts

```diff
@@ -110,5 +151,5 @@ export const pickPlotConfiguration = async (): Promise<
     return
   }
 
-  return { ...templateAndFields, dataFile: file }
+  return { ...templateAndFields }
```

Comment:
This function looks like it's doing many things too. It might be good to break it down into subfunctions like `validateExtensions`, `validateFilesData`... It would also clear this error.

Reply:
Yep, we could also branch the logic depending on the number of files selected.